src/ioc_scan/ioc_scanner.py

```diff
@@ -42,8 +42,10 @@
 few things that should hit:
 GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)
```

Comment:
Just as a thought should we include zsh for macOS since it is the new default?

Reply:
Yes, that's an excellent idea - added in https://github.com/cisagov/ioc-scanner/pull/42/commits/8925967c197821cc3fd2278b578dcbdc3b180543.